includes/wc-legacy-functions.php

```diff
@@ -9,50 +9,12 @@
  *
  * @param int $parent_order_id
  *
- * @throws Exception
- * @return void
+ * @deprecated DOKAN_SINCE
  *
- * @hooked woocommerce_checkout_update_order_meta - 10
+ * @return void
  */
 function dokan_create_sub_order( $parent_order_id ) {
-    if ( dokan_string_to_bool( get_post_meta( $parent_order_id, 'has_sub_order' ) ) === true ) {
-        $args         = [
-            'post_parent' => $parent_order_id,
-            'post_type'   => 'shop_order',
-            'numberposts' => - 1,
-            'post_status' => 'any',
-        ];
-        $child_orders = get_children( $args );
-
-        foreach ( $child_orders as $child ) {
-            wp_delete_post( $child->ID );
-        }
-    }
-
-    $parent_order = new WC_Order( $parent_order_id );
-    $sellers      = dokan_get_sellers_by( $parent_order_id );
-
-    // return if we've only ONE seller
-    if ( count( $sellers ) === 1 ) {
-        $temp      = array_keys( $sellers );
-        $seller_id = reset( $temp );
-        wp_update_post(
-            [
-                'ID'          => $parent_order_id,
-                'post_author' => $seller_id,
-            ]
-        );
-
-        return;
-    }
-
-    // flag it as it has a suborder
-    update_post_meta( $parent_order_id, 'has_sub_order', true );
-
-    // seems like we've got multiple sellers
-    foreach ( $sellers as $seller_id => $seller_products ) {
-        dokan_create_seller_order( $parent_order, $seller_id, $seller_products );
-    }
+    wc_deprecated_function( 'dokan_create_sub_order', '3.7.13', 'dokan()->order->create_sub_order()' );
```

Comment:
Bhai we should add `dokan()->order->create_sub_order()` below `wc_deprecated_function` as well.

Reply:
Those are legacy functions. I don't think we need to keep supporting those functions anymore. we will be removing those functions later on.